include/c_dataset.h

```diff
@@ -49,6 +50,13 @@ extern "C" {
 void* CDataSet(const char* name,
                const size_t name_length);
 
+/*!
+*   \brief C-DataSet destructor
+*   \param name The name of the dataset
```

Comment:
``name `` isn't an input parameter for this function.

Reply:
Good catch! I'll fix that